library/build.gradle

```diff
@@ -25,25 +23,26 @@ android {
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
-
-        kotlinOptions.freeCompilerArgs += ['-module-name', "${GROUP}.${POM_ARTIFACT_ID}"]
+        //kotlinOptions.freeCompilerArgs += ['-module-name', "${GROUP}.${POM_ARTIFACT_ID}"]
```

Comment:
This exists for a reason, it is required for publishing.

Reply:
I have issues on jitpack.io builds